src/platform-includes/getting-started-install/javascript.mdx

```diff
@@ -1,14 +1,27 @@
-```bash {tabTitle: yarn}
-yarn add @sentry/browser
+In order to get started using the Sentry JavaScript SDK, add the following code to the top of your application, before all other scripts:
+
+```html
+<script
+  src="https://js.sentry-cdn.com/___PUBLIC_KEY___.min.js"
+  crossorigin="anonymous"
+></script>
 ```
 
-```bash {tabTitle: npm}
+The Loader Script allows you to configure some SDK features from the Sentry UI, without having to redeploy your application. The [Loader Script documentation](/platforms/javascript/install/loader/) shows more information about how to use it.
```

Comment:
@shanamatthews I updated this to show the loader config on top!

Reply:
this is great, thanks for changing!